src/systems/ec/implementations/vanilla/packages/ec-component-inpage-navigation/ec-component-inpage-navigation.js

```diff
@@ -0,0 +1,221 @@
+/**
+ * Navigation inpage related behaviors.
+ */
+
+import Stickyfill from 'stickyfilljs';
+import gumshoe from 'gumshoejs';
+import { queryOne, queryAll } from '@ecl/ec-base/helpers/dom';
+
+/**
+ * @param {object} options Object containing configuration overrides
+ */
+class InpageNavigation {
```

Comment:
please `export class InpageNavigation` otherwise it won't be part of the vanilla JS build

Reply:
> the focus state is very important
I've tried to use the component with the keyboard and couldn't tell that I was focus the button

@yhuard i took a look at the demo in V1 and i couldn't notice differences regarding this, in the specs we have a generic reference "no changes" in the interaction, so i would assume we have to reproduce what we have in v1, most likely.